plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -231,7 +233,9 @@ protected void verifyConcurrentUpdateFailurePermissible(Exception e)
     protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
     {
         assertThat(e)
-                .hasMessageContaining("Cannot update Iceberg table: supplied previous location does not match current location");
```

Comment:
is `Cannot update Iceberg table: supplied previous location does not match current location"` still part of the error message?

Reply:
it's in the root cause. I've updated the check to also include checking the message of the root cause